core/src/main/java/com/amplifyframework/auth/AuthException.java

```diff
@@ -322,6 +322,35 @@ public UsernameExistsException(Throwable cause) {
         }
     }
 
+    /**
+     * Could not perform the action because user was not found in the system.
```

Comment:
How is this different from `UserNotFoundException`?

Reply:
agreed and now using the UserNotFoundException